### PR TITLE
#1: Implemented BucketFS URL and basic copying.

### LINKED ISSUE #1: Copy files from and to a bucket
# Acceptance Criteria
* Users can copy a local file to a bucket
* Users can copy a file from a bucket to the local file system

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #1.

### CODE REVIEW

File: doc/system_requirments.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# System Requirement Specification BucketFS Java
```
Comment (pj-spoelders) [inline doc/system_requirments.md:1]: File name: 'system_requirments.md' should be 'system_requirements.md'

File: doc/design.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+* `req~gnu-style-command-line-arguments~1`
+
+Needs: impl
+
+# Quality Scenarios
+
+# Risks
+
+# Acknowledgments
```
Comment (pj-spoelders) [inline doc/design.md:139]: Acknowledgements

**Review by pj-spoelders — APPROVED** [review]
Looks good/great to me! Just found 2 small typos in a file name and the documentation.

File: doc/design.md (hunk 2 above)
Comment (pj-spoelders) [reply doc/design.md:139]: Apparently Acknowledgments is also correct, never mind ..

File: doc/system_requirments.md (hunk 1 above)
Comment (redcatbear) [reply doc/system_requirments.md]: Fixed.

**Review by pj-spoelders — APPROVED** [review]

**Review by pj-spoelders — APPROVED** [review]

**Pull request merged**